### DISCUSSION TITLE
How to use REC curve metric?

### DISCUSSION THREAD

**Discussion by Gorbov** [keras-team/autokeras#1624] [Q&A]
May I ask a one example how to use custom metric for ak.StructuredDataRegressor?

**Comment by haifeng-jin** [comment]
https://autokeras.com/tutorial/faq/#how-to-use-customized-metrics-to-select-the-best-model
Please refer to this tutorial.
